nemo/collections/asr/data/data_simulation.py

```diff
@@ -622,7 +671,7 @@ def _build_sentence(
         )
 
         # initialize sentence, text, words, alignments
-        self._sentence = torch.zeros(0, dtype=torch.float64)
+        self._sentence = torch.zeros(0, device=self._device)
```

Comment:
Does it work without `dtype=torch.float64`? It didn't work for me, but maybe it was because I wasn't using the latest pytorch

Reply:
added this